Assets/Scripts/PlayerScripts/PlayerMovement.cs

```diff
@@ -30,5 +42,16 @@ public void UpdateMovementDirection(InputAction.CallbackContext context)
 
 			_playerHealth.ToggleNaturalHealing(_movementDirection.magnitude <= 0.01f);
 		}
+
+		public void Dash(InputAction.CallbackContext context) {
```

Comment:
I feel that the player should be invulnerable during the dash (i.e. can't take damage).